### ISSUE TITLE
Separate tests from implementations

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Cantera/cantera#137]
```
There are several classes which define overloaded constructors intended to 
implement test problems. Having tests mixed in with the implementation pollutes 
the public API and the documentation. The following classes have tests written 
in this manner:

HMWSoln
MargulesVPSSTP
MixedSolventElectrolyte
PhaseCombo_Interaction
RedlichKisterVPSSTP
RedlichKwongMFTP

These constructors should be removed and any useful test code they contain 
incorporated into new or existing tests.

```

Original issue reported on code.google.com by `yarmond` on 8 Jan 2013 at 7:36

**Comment by speth** [comment]
The input files referenced by most of these "tests" have never been included in Cantera, so these have  been marked deprecated (c540e67) and slated for removal after the release of Cantera 2.2. The exception is the one for HMWSoln, which was integrated with an existing test (363d3ec3f0).

**Comment by speth** [comment]
The `MargulesVPSSTP` test was merged with the VCS-LiSi test in d4299fc. The others were removed in f1f10b6.
